packages/build/test/builder/builder.spec.ts

```diff
@@ -1,30 +1,29 @@
 import { VulcanBuilder } from '../../src';
 import * as path from 'path';
-import { IBuildOptions, SchemaReaderType } from '@vulcan-sql/build/models';
-import {
-  PersistentStoreType,
-  SerializerType,
-  TemplateProviderType,
-} from '@vulcan-sql/core';
+import { IBuildOptions } from '@vulcan-sql/build/models';
 
 it('Builder.build should work', async () => {
   // Arrange
   const builder = new VulcanBuilder();
   const options: IBuildOptions = {
-    schemaParser: {
-      reader: SchemaReaderType.LocalFile,
+    'schema-parser': {
+      reader: 'LocalFile',
```

Comment:
The reason I removed the enum for these properties is to avoid casting when using customer extension like below:
```ts
const options: IBuildOptions = {
   'schema-parser': {
      reader: 's3' as SchemaReaderType,
    },
}
```

For avoiding typos in our code, I'll add these enums back, but I prefer to keep the interface the same (string type), so these enums only protect our internal code.

I'll put these enums into `models` folder, and TypeScript is smart enough to know the affinity between string and enums, so we don't need to use `.toString` function to cast them 😄 

This is the final result:
```ts
  const options: IBuildOptions = {
    'schema-parser': {
      reader: SchemaReaderType.LocalFile,
      folderPath: path.resolve(__dirname, 'source'),
    },
    'document-generator': {
      specs: [DocumentGeneratorSpec.oas3],
      folderPath: path.resolve(__dirname),
    },
    artifact: {
      provider: ArtifactBuilderProviderType.LocalFile,
      serializer: ArtifactBuilderSerializerType.JSON,
      filePath: path.resolve(__dirname, 'result.json'),
    },
    template: {
      provider: TemplateProviderType.LocalFile,
      folderPath: path.resolve(__dirname, 'source'),
    },
    extensions: {},
  };
```
4b905db1d0b35d594bc8c32fb1a9df3dfb00d752